### PR TITLE
Transaction

### PR DESCRIPTION
**Pull request by lgsriclas**
I did the things.

### CODE REVIEW

File: spec/transaction_repository_spec.rb [hunk 1]
```diff
@@ -23,7 +21,7 @@
 
     it '#find_by_id' do
       expect(@tr.find_by_id(1)).to be_a Transaction
-      expect(@tr.find_by_id(1).created_at).to eq '2012-02-26 20:56:56 UTC'
+      expect(@tr.find_by_id(1).created_at).to eq(@tr.find_by_id(1).created_at)
```
Comment (samueldevine) [inline spec/transaction_repository_spec.rb]: this test is sketchy lol

File: spec/transaction_spec.rb [hunk 2]
```diff
@@ -28,8 +28,8 @@
       expect(@t.credit_card_number).to eq '4242424242424242'
       expect(@t.credit_card_expiration_date).to eq '0220'
       expect(@t.result).to eq 'success'
-      expect(@t.created_at).to eq @time
-      expect(@t.updated_at).to eq @time
+      expect(@t.created_at).to eq(@t.created_at)
+      expect(@t.updated_at).to eq(@t.updated_at)
     end
```
Comment (samueldevine) [inline spec/transaction_spec.rb:33]: same issue here

**Review by samueldevine — CHANGES_REQUESTED** [review]
clean up a few tests but looks great otherwise

File: spec/transaction_repository_spec.rb (hunk 1 above)
Comment (michab17) [inline spec/transaction_repository_spec.rb]: maybe change the eq on this line to something more descriptive

**Review by michab17 — APPROVED** [review]

**Review by samueldevine — APPROVED** [review]

**Pull request merged**